### PR TITLE
update lodash

### PR DESCRIPTION
**Pull request by dickeyxxx**

### CODE REVIEW

**Comment by bjeanes** [comment]
@dickeyxxx seems to include the Circle CI commit too

**Comment by dickeyxxx** [comment]
rebased

File: commands/addons.js [hunk 1]
```diff
@@ -89,7 +89,7 @@ function* addonGetter(api, app) {
 }
 
 function displayAll(addons) {
-    addons = _.sortByAll(addons, 'app.name', 'plan.name', 'addon.name');
+    addons = _.sortBy(addons, 'app.name', 'plan.name', 'addon.name');
```
Comment (bjeanes) [inline commands/addons.js]: Maybe it accepts both, but the docs imply that `_.sortBy` takes an array of fields as the second argument when sorting by multiple fields, not var args

Comment (bjeanes) [inline commands/addons.js]: Er never mind I see they have at least one example that doesn't use an array

**Comment by bjeanes** [comment]
LGTM

File: commands/addons.js (hunk 1 above)
Comment (dickeyxxx) [inline commands/addons.js]: it takes both, noted with the ellipses:
screen shot 2016-03-31 at 4 27 48 pm

Comment (bjeanes) [inline commands/addons.js]: Gah ah got it. Thanks

**Pull request merged**